packages/claroty_ctd/manifest.yml

```
@@ -1,7 +1,7 @@
 format_version: 3.1.4
 name: claroty_ctd
 title: Claroty CTD
-version: 0.1.1
+version: 0.2.1
```

```suggestion
version: 0.1.2
```
Can be a patch release because it's a bug fix rather than new functionality.